### ISSUE TITLE
Release TAG is very hard to follow...

### ISSUE THREAD

**Issue by MatMoul** [Azure/WALinuxAgent#2135]
I'm the maintainer of the ArchLinux walinuxagent package and I try to follow the release tag to update the package but it's very hard to follow it..
Looking at this page : https://github.com/Azure/WALinuxAgent/releases
At this time, V2.2.51 is the latest release but the title say "this release was not completed in all regions"
How can I be sure that a release is ready?

**Comment by pgombar** [comment]
Hi @MatMoul, we update the title as the release is completed in all product regions. I have just updated the [v2.2.35]( https://github.com/Azure/WALinuxAgent/releases/tag/v2.2.53) tag since we just completed the release. This is the latest stable version of the agent and you can update to this version.

Previously to this update, you would go to the release page and see the first release tag that doesn't contain anything like "will update once release completed".

**Comment by MatMoul** [comment]
@pgombar Thanks for the reply, package updated to 2.2.53 and the latest tag is verry clean ;-)
Watching for the next release...
